README.md

```diff
@@ -1,138 +1,49 @@
 
 ## AutoGVP Workflow  
 <img src="https://github.com/diskin-lab-chop/pathogenicity-assessment/blob/6848a369885fb9e03e75ec82afc97379457c0275/figures/germline-pathogenecity_flow.png" align="center" width = "600">
 
+For more detailed instructions, please visit the [user guide](https://github.com/diskin-lab-chop/AutoGVP/wiki/User-Guide) on our wiki.
+
 ## Clone the AutoGVP repository
 ```bash
 git clone https://github.com/diskin-lab-chop/AutoGVP.git
 ```
 
 ## Docker set-up
-
-### docker pull
+### docker pull, run, and execute
 ```bash
 docker pull pgc-images.sbgenomics.com/naqvia/autogvp:latest
 ```
 cd to your clone of `AutoGVP`
-
-### docker run
 Replace <CONTAINER_NAME> with any name and run the command below:
 ```
 docker run --platform linux/amd64 --name <CONTAINER_NAME> -d -v $PWD:/home/rstudio/AutoGVP pgc-images.sbgenomics.com/naqvia/autogvp:latest
-```
-### docker execute
-```bash
 docker exec -ti <CONTAINER_NAME> bash
 ```
-
 ### cd to AutoGVP directory within docker
 ```bash
 cd /home/rstudio/AutoGVP
 ```
 
-## How to Run
-
-### CAVATICA input ###
-1. Run the [Kids First Germline Annotation Workflow](https://github.com/kids-first/kf-germline-workflow/blob/v0.4.4/docs/GERMLINE_SNV_ANNOT_README.md) first.
-This workflow currently annotates variants with ClinVar (2022-05-07).
-2. Run the [Pathogenicity Preprocessing Workflow](https://github.com/d3b-center/D3b-Pathogenicity-Preprocessing), which performs ANNOVAR with InterVar and AutoPVS1 annotations.
-3. Download required db files, including variant submissions files (`variant_summary.txt` and `submission_summary.txt`) using download_db_files.sh (see below).
-4. Run `select-clinVar-submissions.R` within `scripts/` to consolidate ClinVar submission calls prior to running AutoGVP wrapper script. This step can be ran once and the `ClinVar-selected-submissions.tsv` file could be used for subsequent runs if clinVar versions remain the same. 
-5. Run AutoGVP wrapper script using `--workflow="cavatica"`
+## How to Run AutoGVP
 
 AutoGVP Requirements (recommended to place all in the `data/` folder):
 - VEP-, ANNOVAR-, and ClinVar annotated VCF file (`*VEP.vcf`)
 - ANNOVAR multianno file (`*hg38_multianno.txt`)
 - InterVar file (`*intervar.hg38_multianno.txt.intervar`)
 - AutoPVS1 file (`*autopvs1.txt`)
 - Variant submissions file (`ClinVar-selected-submissions.tsv` generated by `select-clinVar-submissions.R`)
+- ClinVar VCF (`clinvar_yyyymmdd.vcf.gz` optional user input or `clinvar.vcf.gz` will be downloaded with `download_db_files.sh`) - custom workflow only
 
-
-Example run:
-
-Download ClinVar files:
-
-```bash
-bash scripts/download_db_files.sh
-```
-
-Run `select-clinVar-submissions.R`:
-
+### Custom workflow example run
+1. [Prepare input files](https://github.com/diskin-lab-chop/AutoGVP/wiki/User-Guide#custom-input-workflow---step-by-step) by running VEP, ANNOVAR, InterVar, and AutoPVS1.
+2. Run `select-clinVar-submissions.R`:
```

Comment:
should we also note about where to obtain --conceptID_list (if they want to generate their own) and for --conflct_res there is also option of "most_recent"

Reply:
@jungkim2 I added the options but can you send me a link to the concept id list or file used, i will add it to the wiki, then link to it from here